koptional/src/main/kotlin/com/gojuno/koptional/Optional.kt

```diff
@@ -9,6 +9,8 @@ sealed class Optional<out T : Any> {
 }
 
 data class Some<out T : Any>(val value: T) : Optional<T>()
-object None : Optional<Nothing>()
+object None : Optional<Nothing>() {
```

Comment:
nit: maybe add an empty line between classes now? 

With additional code it now looks a little bit crumpled :)